com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/PathTracing/Shaders/PathTracingMain.raytrace

```diff
@@ -53,6 +57,13 @@ void MissMaterial(inout PathIntersection pathIntersection : SV_RayPayload)
     ApplyFogAttenuation(WorldRayOrigin(), WorldRayDirection(), pathIntersection.value);
 }
 
+float2 UniformPointWithinCircle(float radius, float u1, float u2)
```

Comment:
There is already a SampleDiskUniform() in Sampling.hlsl, that does exactly that.
(You can then multiply by the radius in the caller)